src/heretic/r_things.c

```diff
@@ -752,7 +752,24 @@ void R_AddSprites(sector_t * sec)
 
 
     for (thing = sec->thinglist; thing; thing = thing->snext)
+    {
+        // [crispy] draw base frame and translucent current frame for mummyleader attack
+        if (crispy->translucency & TRANSLUCENCY_MISSILE)
+        {
+            if (thing->info->doomednum == 45 && thing->frame == 32792)
```

Comment:
Could you check for the sprite here, instead of the (cryptic) `doomednum`, please?

Reply:
Or the state? Should be `S_MUMMYL_ATK2`.